testsuite/testsuite_impl/dynamic_single_node_nginx_test_.go

```diff
@@ -16,13 +16,17 @@ import (
 	"time"
 )
 
-type SingleNodeExampleTest struct {
+type DynamicSingleNodeExampleTest struct {
```

Comment:
Renamed this just to emphasize that it's dynamic, which is something I wanted to do since I wrote the tutorial